.gitignore

```diff
@@ -1 +1,2 @@
 doc/_build/
+_build.*
```

Comment:
What files are these? I wasn't expecting this package to be buildable, since it doesn't have a SConscript...

Reply:
the eupspkg system always tries to do something even if that's copying a bunch of files over.